<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diskfind</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Device</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="DevComboBox" name="devSelectCombo"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="devSizeLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="analyseFS">
            <property name="toolTip">
             <string>Tries to retrieve file path to file with searched text.
This option can SIGNIFICANTLY slowdown searching process on large disks.</string>
            </property>
            <property name="text">
             <string>Analyse filesystem</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Block number</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="sectorNumberSpin">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="showSectorButton">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Search parameters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="1">
             <widget class="CheckBoxList" name="cbxEncodings"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Encodings</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Document types</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="CheckBoxList" name="cbxDocTypes"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Search type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="1">
             <widget class="QLineEdit" name="searchTextEdit"/>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="chxUsePhrase">
              <property name="text">
               <string>by phrase</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="chxUseDictionary">
              <property name="text">
               <string>by dictionaries</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="CheckBoxList" name="cbxDictionary"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="dictLine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="dictContentsWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="1">
                <widget class="QLabel" name="dictLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Contents of dictionary</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListWidget" name="wordsList">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAsNeeded</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>wordsList</zorder>
            <zorder></zorder>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editDictButton">
         <property name="text">
          <string>Edit dictionary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addDictButton">
         <property name="text">
          <string>Add dictionary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Display encoding</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="dataCoding"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="toggleSearchButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="hexTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string notr="true">0x0000</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0010</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0020</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0030</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0040</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0050</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0060</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0070</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0080</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0090</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x00A0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x00B0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x00C0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x00D0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x00E0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x00F0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0100</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0110</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0120</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0130</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0140</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0150</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0160</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0170</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0180</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x0190</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x01A0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x01B0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x01C0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x01D0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x01E0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">0x01F0</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string notr="true">0</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true"/>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">6</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">7</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true"/>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">8</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">9</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">A</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">B</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true"/>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">C</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">D</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">E</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">F</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true"/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
      </column>
      <item row="0" column="20">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBoxList</class>
   <extends>QComboBox</extends>
   <header location="global">checkboxlist.h</header>
  </customwidget>
  <customwidget>
   <class>DevComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">devcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>devSelectCombo</tabstop>
  <tabstop>analyseFS</tabstop>
  <tabstop>sectorNumberSpin</tabstop>
  <tabstop>showSectorButton</tabstop>
  <tabstop>cbxEncodings</tabstop>
  <tabstop>cbxDocTypes</tabstop>
  <tabstop>chxUsePhrase</tabstop>
  <tabstop>searchTextEdit</tabstop>
  <tabstop>chxUseDictionary</tabstop>
  <tabstop>cbxDictionary</tabstop>
  <tabstop>wordsList</tabstop>
  <tabstop>dataCoding</tabstop>
  <tabstop>toggleSearchButton</tabstop>
  <tabstop>hexTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
